src/components/SearchResultsList/SearchResultsList.js

```diff
@@ -80,12 +81,27 @@ const SearchResultsList = ({
         : authority.authRefType;
     },
     headingRef: (authority) => (
-      <TextLink
-        to={`${match.path}/authorities/${authority.id}`}
-        className={authority.isAnchor ? css.anchorLink : null}
-      >
-        {authority.headingRef}
-      </TextLink>
+      authority.isAnchor
```

Comment:
`isAnchor` will exist for both exact and not exact matches, so we should define a new property `isExactMatch` in `<BrowseRoute> formattedAuthoritiesForView`.
If `authorityItem.authority` is not undefined then it's an exact match
And then display no match element if `isAnchor && !isExactMatch`

Reply:
You are right, thanks! I don't know how I didn't notice it.